xlators/cluster/ec/src/ec-inode-read.c

```diff
@@ -1353,6 +1353,7 @@ ec_manager_readv(ec_fop_data_t *fop, int32_t state)
 {
     ec_cbk_data_t *cbk;
     ec_t *ec = fop->xl->private;
+    ec_inode_t* ctx = __ec_inode_get(fop->fd->inode, fop->xl);
```

Comment:
Maybe you should write two functions, `ec_inode_readmask_{g,s}et()` and use them to get/set readmask.